neo/Network/P2P/Payloads/VersionPayload.cs

```diff
@@ -1,42 +1,53 @@
 ﻿using Neo.IO;
+using Neo.IO.Caching;
+using Neo.Network.P2P.Capabilities;
 using System;
+using System.Collections.Generic;
 using System.IO;
+using System.Linq;
 
 namespace Neo.Network.P2P.Payloads
 {
     public class VersionPayload : ISerializable
     {
+        /// <summary>
+        /// Reflection cache for ConsensusMessageType
+        /// </summary>
+        private static readonly ReflectionCache<byte> ReflectionCache = ReflectionCache<byte>.CreateFromEnum<NodeCapabilities>();
+
+        const int MaxCapabilities = 32;
+
         public uint Magic;
         public uint Version;
         public VersionServices Services;
         public uint Timestamp;
-        public ushort Port;
         public uint Nonce;
         public string UserAgent;
         public uint StartHeight;
```

Comment:
It could be moved to a new Capability, because it could be a light node without chain, something like `BlockchainCapability` -> Height
What do you think @erikzhang  ?

Reply:
I think this is a feature of `VersionServices.NodeNetwork`.

Reply:
For me, is something optional, because you can be a light node, we should move this to a new capability

Reply:
Then we can remove `VersionServices`.